c/tree.c

```diff
@@ -215,7 +218,7 @@ void insert_network(MMDBW_tree_s *tree, const char *ipstr,
     free_network(&network);
 
     if (MMDBW_SUCCESS != status) {
-        croak("%s (%s)", status_error_message(status), ipstr);
+        croak("%s (when inserting %s/%" PRIu8 ")", status_error_message(status), ipstr);
```

Comment:
I think this call to `croak` is missing an argument. There are 3 sprintf format bits, but only two values.